### ISSUE TITLE
strHdrType DB field does not exist in Kodi 19

### ISSUE THREAD

**Issue by oddstr13** [jellyfin/jellyfin-kodi#852]
I needed to resync my whole library and i'm getting a failure too. It just sits at 0% stuck on the first episode of the fist show in my library. 

Jellyfin 10.8.13
JfK 1.0.1
Kodi 19.5

````
2024-04-09 10:24:09.517 T:3746     INFO <general>: JELLYFIN.jellyfin_kodi.objects.tvshows -> INFO::jellyfin_kodi/objects/tvshows.py:311 EpisodeId 1 missing from kodi. repairing the entry.
2024-04-09 10:24:09.575 T:3746     INFO <general>: JELLYFIN.jellyfin_kodi.objects.tvshows -> DEBUG::jellyfin_kodi/objects/tvshows.py:404 Setting episode pathParentId, episode 45b2ab24b3a9d76867094b5a4a4d8144, title Countdown, pathId 4, pathParentId None
2024-04-09 10:24:09.583 T:3746     INFO <general>: JELLYFIN.jellyfin_kodi.objects.tvshows -> DEBUG::jellyfin_kodi/objects/tvshows.py:407 ADD episode [4/2] 45b2ab24b3a9d76867094b5a4a4d8144: Countdown
2024-04-09 10:24:09.592 T:3746     INFO <general>: JELLYFIN.jellyfin_kodi.database -> ERROR::jellyfin_kodi/database/__init__.py:165 type: <class 'sqlite3.OperationalError'> value: table streamdetails has no column named strHdrType
2024-04-09 10:24:09.596 T:3746     INFO <general>: JELLYFIN.jellyfin_kodi.database -> DEBUG::jellyfin_kodi/database/__init__.py:169 [jellyfin] 2 rows updated.
2024-04-09 10:24:09.605 T:3746     INFO <general>: JELLYFIN.jellyfin_kodi.database -> DEBUG::jellyfin_kodi/database/__init__.py:172 ---<[ database: jellyfin ] 68548680
2024-04-09 10:24:09.611 T:3746     INFO <general>: JELLYFIN.jellyfin_kodi.database -> ERROR::jellyfin_kodi/database/__init__.py:165 type: <class 'sqlite3.OperationalError'> value: table streamdetails has no column named strHdrType
2024-04-09 10:24:09.615 T:3746     INFO <general>: JELLYFIN.jellyfin_kodi.database -> DEBUG::jellyfin_kodi/database/__init__.py:169 [video] 81 rows updated.
2024-04-09 10:24:09.629 T:3746     INFO <general>: JELLYFIN.jellyfin_kodi.database -> DEBUG::jellyfin_kodi/database/__init__.py:172 ---<[ database: video ] 70197544
````

_Originally posted by @gurg1e in https://github.com/jellyfin/jellyfin-kodi/issues/849#issuecomment-2043935680_

**Comment by oddstr13** [comment]
This DB field is new to Kodi 19, I didn't catch it because I am on Kodi 20. xbmc/xbmc#19983

The suggested workaround is to upgrade to Kodi 20 or newer, but we may want to consider fixing this particular issue in the plugin.

That said, Kodi 21 was just released, which means Kodi 19 is technically out of support for this project, and I don't want to normalize maintaining ancient versions, at least not ones I personally don't use any more.

**Comment by gurg1e** [comment]
Thanks for the dedicated ticket and the extra details. Last time I tried upgrading to Kodi 20.x I had some issues with the TV tuner (from memory). I will investigate upgrading to 20 or 21. 

Cheers

**Comment by oddstr13** [comment]
@gurg1e Should be fixed with v1.0.2, if you haven't gotten Kodi 20/21 working yet

**Comment by gurg1e** [comment]
Wow such a quick fix, thanks! 

I ended up reinstalling my whole system with the latest Kodi 21. I was overdue a fresh install anyway. Thanks again.
